src/msg/async/dpdk/dpdk_rte.cc

```diff
@@ -123,6 +125,21 @@ namespace dpdk {
         args.push_back(string2vector("--no-huge"));
       }
 
+      if (!cct->_conf->ms_dpdk_port_options.empty()) {
+        std::list<std::string> port_list;
+        get_str_list(cct->_conf.get_val<std::string>("ms_dpdk_port_options"), " ", port_list);
+        for (auto &i : port_list) {
+          size_t equal = i.find('=');
+          if (equal != std::string::npos) {
+             const std::string &key = i.substr(0,equal);
+             equal++;
+             const std::string &value = i.substr(equal);
+             args.push_back(string2vector(key));
+             args.push_back(string2vector(value));
+           } else
+             args.push_back(string2vector(i));
+        }
+      }
```

Comment:
```suggestion
      for (auto [key, val] : get_str_map(cct->_conf->ms_dpdk_port_options)) {
        args.push_back(string2vector(key));
        if (!val.empty()) {
          args.push_back(string2vector(val));
        }
      }
```

Reply:
We cannot use Map because there is a repetitive key, such as designating two network ports for Bond.

Reply:
We cannot use Map because there is a repetitive key, such as designating two network ports for Bond.

Reply:
We cannot use Map because there is a repetitive key, such as designating two network ports for Bond.

> For documentation purposes, please use allowlist.

Ok